native/packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNavSelectedIndicator.js

```diff
@@ -0,0 +1,100 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {
+  Animated,
+  Platform,
+  StyleSheet,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import { withTheme } from 'react-native-bpk-theming';
+
+import { THEMING_ATTRIBUTE, themePropType } from './theming';
+
+const ANIMATION_DURATION = 200; // milliseconds.
```

Comment:
Use http://backpack.prod.aws.skyscnr.com/tokens/animation

Reply:
Just tried this and it's not possible with the current tokens as they're strings -- `"200ms"` as opposed to `200`.

I'll do another PR in the future to change the formatting of the tokens and then update this, I think it's out of scope for this PR.